src/ui/server/controllers/insurance/business/companies-house-number/index.test.ts

```diff
@@ -231,7 +231,7 @@ describe('controllers/insurance/business/companies-house-number', () => {
 
     describe('when there is no application', () => {
       beforeEach(() => {
-        res.locals = { csrfToken: '1234' };
+        res.locals = { csrfToken: '1234', SRI: {}, meta: {} };
```

Comment:
We should create a `mock-locals` in tests mocks, not a blocker. Tech improvement just making a note